### PR TITLE
Removes: un-used resources using the Android studio tool 1/2

### PR DESCRIPTION
**Pull request by AjeshRPai**
Fixes #
## Description
This PR deletes the unused resources in the code using [Remove Unused Resources tool ](https://stackoverflow.com/questions/6373482/remove-all-unused-resources-from-an-android-project)in the Android Studio

## To Test:
1. Smoke test the JP And WP app and verify that everything is working as expected 
2. Verify that the classess, resources and files deleted are unused. 

## Regression Notes

1. Potential unintended areas of impact
App is not working as expected

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and Smoke testing 

3. What automated tests I added (or what prevented me from doing so)
Nothing

PR submission checklist:
- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist:
- [x] Portrait and landscape orientations.
- [x] Light and dark modes.
- [x] Fonts: Larger, smaller and bold text.
- [x] High contrast.
- [x] Talkback.
- [x] Languages with large words or with letters/accents not frequently used in English.
- [x] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [x] Large and small screen sizes. (Tablet and smaller phones)
- [x] Multi-tasking: Split screen and Pop-up view. (Android 10 or higher)

### CODE REVIEW

**Review by zwarm — APPROVED** [review]
@AjeshRPai - I spent a good hour going through the features. I feel that I can approve this change; however I would love it if someone can give it a whirl as it's such a big change and no real testing steps. 👍

**Comment by AjeshRPai** [comment]
Thanks @zwarm for the review. 🙌🏼 . You are right. I will wait until some one else also takes a look at it and gives me a 👍🏼  to merge. 

Also, I came across one more class after your review which seems to be unused. See the commit https://github.com/wordpress-mobile/WordPress-Android/pull/19788/commits/5b177d59f8b1a93e1e7dda1ae51ca77230284ad7

Feel free to re-review if you think its necessary.

**Comment by AjeshRPai** [comment]
Hey @irfano @ravishanker 

Since you left a 👍🏼  in the Slack message, I assume you both want to review the PR. Hence, I am adding both of you as reviewers.

I am waiting for one more approval to merge this PR. So feel free to ignore this PR if there is two approval. 

Thank you in advance for the review.

File: WordPress/src/main/java/org/wordpress/android/widgets/FlowLayout.java
Comment (irfano) [inline WordPress/src/main/java/org/wordpress/android/widgets/FlowLayout.java:29]: We can also remove `FlowLayout` and `FlowLayout_Layout` in `attrs.xml`

**Review by irfano — APPROVED** [review]
I tested basic features manually. LGTM! 👍🏻 Thanks for taking care of this.

File: WordPress/src/main/java/org/wordpress/android/widgets/FlowLayout.java
Comment (AjeshRPai) [reply WordPress/src/main/java/org/wordpress/android/widgets/FlowLayout.java:29]: Thanks @irfano for pointing it out. Removed both in [fcc6fc3](https://github.com/wordpress-mobile/WordPress-Android/pull/19788/commits/fcc6fc3b140fced5c2188cc1ad2cb7e3ff473979)

**Pull request merged**
